packages/server/src/resolvers/Mutation.ts

```diff
@@ -4,6 +4,14 @@ export const Mutation: MutationResolvers = {
   resetDocker: async (parent, args, { docker }) => {
     return await docker.reset();
   },
+  resetVolume: async (parent, args, { docker, dataSources }) => {
+    const [volume, dataset, project] = await Promise.all([
+      docker.resetVolume(),
+      dataSources.datasetService.reset(),
+      dataSources.projectService.reset()
```

Comment:
I think you can use
```suggestion
      sequelize.sync({ force: true })
```
https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html#instance-method-sync

to really reset the entire thing without leaving anything behind.